src/quatrex/coulomb_screening/polarization.py

```diff
@@ -114,20 +125,40 @@ def compute(
             for start, end in zip(batch_displacements, batch_displacements[1:]):
                 batch = slice(start, end)
 
+                synchronize_device()
+                time_fft -= time.perf_counter()
```

Comment:
Polarization and Self-energies are very slow when measuring time per batch. Therefore, we should go back to measuring at a higher level and let lower-level profiling to nsys/ncu.